<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>260</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Терминал очереди МАНО &quot;ЛДЦ&quot;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QPushButton" name="btnBook">
      <property name="text">
       <string>Плановая запись</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/book.svg</normaloff>:/icons/icons/book.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDisp">
      <property name="text">
       <string>Диспансеризация</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/hospital.svg</normaloff>:/icons/icons/hospital.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReceipt">
      <property name="text">
       <string>Выписка льготного рецепта</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/medicine.svg</normaloff>:/icons/icons/medicine.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEmergency">
      <property name="text">
       <string>На приём по острому состоянию</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/emergency.svg</normaloff>:/icons/icons/emergency.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnReport">
      <property name="text">
       <string>Санаторно-курортная карта, операция, справка, выписка</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/report.svg</normaloff>:/icons/icons/report.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnVaccine">
      <property name="text">
       <string>Вакцинация (клещ, грипп)</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/vaccine.svg</normaloff>:/icons/icons/vaccine.svg</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOther">
      <property name="text">
       <string>Прочее (прикрепление, открепление, копия амбулаторной карты)</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/icons/icons/question.svg</normaloff>:/icons/icons/question.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
